operators/src/mock/mock_raster_source.rs

```diff
@@ -171,7 +171,7 @@ impl<T: Pixel> OperatorDatasets for MockRasterSource<T> {
 ///
 /// TODO: implementation is done with `paste!`, but we can use `core::concat_idents` once its stable
 ///
-/// ```no_run
+/// ```compile_fail
```

Comment:
wouldn't 

>\`\`\`ignore

be more suitable?

Reply:
[d2931b8](https://github.com/geo-engine/geoengine/pull/539/commits/d2931b84c4c0ab87267819a4bde9f8147bed727d)